charts/cache/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: kcp
+description: A prototype of a multi-tenant Kubernetes control plane for workloads on many clusters
+
+# version information
+version: 0.4.4
```

Comment:
I'm not 100% confident on this, but I think all the charts should have their own versioning (so this should be `0.1.0`). Else we would be bumping all chart versions whenever we update _one_ chart.